baseplate/lib/message_queue.py

```diff
@@ -166,7 +294,14 @@ def queue_tool() -> None:
 
     args = parser.parse_args()
 
-    queue = MessageQueue(args.queue_name, args.max_messages, args.max_message_size)
+    if args.queue_type == QueueType.IN_MEMORY.value:
+        # Start a remote queue, which connects to a Thrift server
+        # that manages an in-memory queue
+        queue = RemoteMessageQueue(args.queue_name, args.max_messages)
+    else:
```

Comment:
So the default value is already encoded in the add_argument function, if you change it in furture you would have to change this statetment too. Might be worth writing a helper function which takes a QueueType and returns the correct queue implementation then you don't need to worry about that?